.github/workflows/javascript_tests.yml

```
@@ -12,6 +12,10 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '14' # # Should match what's in our Dockerfile
+      - uses: actions/cache@v2
+        with:
+          path: '~/.npm'
```

TIL! This is the preferred way to cache node_modules.

Although looking at the times, it doesn't look like this made too much difference :/ `npm install` still takes ~1.2min, even with a cache hit.

According to the docs:

> It is not recommended to cache node_modules, as it can break across Node versions and won't work with npm ci
> https://github.com/actions/cache/blob/main/examples.md#node---npm

Well, we pin the node version, and we don't update it that often anyways. And we don't use `npm ci` specifically to try to get faster CI flows. So let's try caching `node_modules` directly.

```suggestion
          # Caching node_modules isn't recommended because it can break across
          # Node versions and won't work with npm ci (See https://github.com/actions/cache/blob/main/examples.md#node---npm )
          # But we pin the node version, and we don't update it that often anyways. And 
          # we don't use `npm ci` specifically to try to get faster CI flows. So caching
          # `node_modules` directly.

          path: '**/node_modules'
```